### PR TITLE
Add support for enabling slurmdbd

### PR DESCRIPTION
**Pull request by jovial**

### CODE REVIEW

File: molecule/test4/verify.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+
+- name: Check slurm hostlist
+  hosts: testohpc_login
+  tasks:
+  - name: Submit a test job
+    command: sbatch --wrap hostname --parsable
+    register: job
+
+  - name: Allow sum time for the job to complete
```
Comment (jovial) [inline molecule/test4/verify.yml:10]:
```suggestion
  - name: Allow some time for the job to complete
```

File: README.md [hunk 2]
```diff
@@ -8,25 +8,42 @@ As a role it must be used from a playbook, for which a simple example is given b
...
+
+`openhpc_enable`:
+* `control`: whether to enable control host
+* `database`: whether to enable slurmdbd
+* `batch`: whether to enable compute nodes
+* `runtime`: whether to enable OpenHPC runtime
+* `drain`: whether to drain compute nodes
+* `resume`: whether to resume compute nodes
+
+`openhpc_slurmdbd_host`: Where to deploy slurmdbd if are using this role to deploy slurmdbd, otherwise where
```
Comment (sjpb) [inline README.md:32]: Needs to specifiy that default is `openhpc_slurm_control_host`

File: README.md [hunk 3]
```diff
@@ -35,17 +52,66 @@ package in the image.
...
-* `batch`: whether to enable compute nodes
-* `runtime`: whether to enable OpenHPC runtime
-* `drain`: whether to drain compute nodes
-* `resume`: whether to resume compute nodes
+By default, the accounting plugin will use the `accounting_storage/filetxt` storage type. However,
+this only supports a subset of `sacct` commands.
+
+To deploy and configure `slurmdbd`:
+
+* Configure a mariadb or mysql server as documented in the slurm accounting (documentation)[https://slurm.schedmd.com/accounting.html]
```
Comment (sjpb) [inline README.md:62]: From this and the following point, it's not clear to me if they have to be the same node? If so I'd suggest replacing both with:
 
- Configure a mariadb or mysql server as described in the slurm accounting [documentation](https://slurm.schedmd.com/accounting.html) on one of the nodes in your inventory and set `openhpc_enable.database` to `true` for this node.

If not, just fix the doc link markup/works as above anyway!

**Review by sjpb — CHANGES_REQUESTED** [review]

**Comment by jovial** [comment]
Thanks for the review. Think I've addressed everything now.

File: README.md [hunk 4]
```diff
@@ -37,19 +52,66 @@ package in the image.
...
-* `drain`: whether to drain compute nodes
-* `resume`: whether to resume compute nodes
+By default, the accounting plugin will use the `accounting_storage/filetxt` storage type. However,
+this only supports a subset of `sacct` commands.
 
-`openhpc_slurm_configless`: Optional, default False. If True then slurm's ["configless" mode](https://slurm.schedmd.com/configless_slurm.html) is used. **NB: Requires Centos8/OpenHPC v2.**
+To deploy and configure `slurmdbd`:
+
+* Configure a mariadb or mysql server as described in the slurm accounting [documentation](https://slurm.schedmd.com/accounting.html) on one of the nodes in your inventory and set `openhpc_enable.database `to `true` for this node.
+* Set `openhpc_enable.database` to `true` for one of the nodes in your inventory
```
Comment (sjpb) [inline README.md:63]: Duplicates above line.

**Review by sjpb — CHANGES_REQUESTED** [review]
Needs one line deleting in README

File: README.md (hunk 4 above)
Comment (jovial) [inline README.md:63]:
```suggestion
```

Comment (jovial) [reply README.md]: Good spot, thanks.

**Review by sjpb — APPROVED** [review]

**Pull request merged**